tests/suite/custom_resources_utils.py

```diff
@@ -287,6 +351,7 @@ def delete_ap_policy(custom_objects: CustomObjectsApi, name, namespace) -> None:
         "appolicies",
         name,
     )
+    time.sleep(3)
```

Comment:
yes, otherwise sometimes can cause conflict in https://github.com/nginxinc/kubernetes-ingress/pull/1352/files#diff-33e0fbe3677c25422f841dfc8ff08d5614792d234c6349481c34c8e2f43f4c0aR279